mep/books/models.py

```diff
@@ -313,6 +313,19 @@ class Work(TrackChangesModel, Notable, ModelIndexable, EventSetMixin):
     genres = models.ManyToManyField(
         Genre, blank=True, help_text="Genre(s) from OCLC record"
     )
+
+    # top level category
```

Comment:
if you use this notation, sphinx should pick it up as a docstring for the attribute

```suggestion
    #: top level category
```